packages/main/src/Tokenizer.ts

```diff
@@ -137,10 +81,66 @@ const metadata = {
  * @usestextcontent
  * @private
  */
+@customElement("ui5-tokenizer")
+@languageAware
+
+@event("token-delete", {
+	detail: {
+		ref: { type: HTMLElement },
+	},
+})
+
+@event("show-more-items-press", {
+	detail: {
+		ref: { type: HTMLElement },
+	},
+})
+
 class Tokenizer extends UI5Element {
-	static get metadata() {
-		return metadata;
-	}
+	@property({ type: Boolean })
+	showMore!: boolean;
+
+	@property({ type: Boolean })
+	disabled!: boolean;
+
+	/**
+	 * Indicates if the tokenizer should show all tokens or n more label instead
+	 *
+	 * @private
+	 */
+	@property({ type: Boolean })
+	expanded!: boolean;
+
+	@property({ type: Object })
+	morePopoverOpener!: Tokenizer;
+
+	@property({ validator: Integer })
+	popoverMinWidth!: number;
```

Comment:
validator without default value may result to undefined so change ! to ?

Reply:
Done